cmd/create/cluster/cmd.go

```diff
@@ -418,7 +418,23 @@ func run(cmd *cobra.Command, _ []string) {
 		mapSubnetToAZ := make(map[string]string)
 		mapAZCreated := make(map[string]bool)
 		options := make([]string, len(subnets))
-		optionDefaults := make([]string, len(subnetIDs))
+		defaultOptions := make([]string, len(subnetIDs))
+
+		// Verify subnets provided exist.
+		if subnetsProvided {
+			for _, subnetArg := range subnetIDs {
+				verifiedSubnet := false
+				for _, subnet := range subnets {
+					if awssdk.StringValue(subnet.SubnetId) == subnetArg {
+						verifiedSubnet = true
+					}
+				}
```

Comment:
Can we do this check inside the next loop? We already have the same loop doing
```
if subnetsProvided {
  for _, subnetArg := range subnetIDs {
// ...
```
It's the exact same loop, let's reuse it!

Reply:
I wanted to error early..